### PR TITLE
Add missing builtins

### PR DESCRIPTION
**Pull request by AmjadHD**
* `GC_enable`
* `GC_enableMarkAndSweep`
* `nimGC_setStackBottom`
* `nimThreadProcWrapperBody`

### CODE REVIEW

**Comment by Araq** [comment]
Oh god nooooo, instead remove the existing builtins, it makes no sense to highlight them differently and it's super annoying. All these syntax highlighters that don't understand what Nim's keywords are. They are all listed in the manual, there are no others.

**Comment by AmjadHD** [comment]
You are right, I didn't notice that other syntaxes do not highlight builtins differently. Sublime Text syntaxes do. So God-willing I'll remove them from Github's grammars (tmLanguage) only.

File: Syntaxes/Nim SCF.tmLanguage [hunk 1]
```diff
@@ -371,7 +371,7 @@
...
...?:Ex)?|handle|high|incl?|insert|instantiationInfo|internalNew |isNil|isNotForeign|iterToProc|joinThreads?|len|lines|locals|low|max|min |m?(?:items|pairs)|move(?:Mem)? |new(?:Seq(?:OfCap|Uninitialized)?|String(?:OfCap)?|Exception|WideCString)? |onFailedAssert|onThreadDestruction|open|ord|peek|pinToCpu|pop(?:GcFrame)?|pred |prepareMutation|procCall|protect|pushGcFrame|quit|raiseAssert|raw(?:Env|Proc) |ready|read(?:All|Buffer|Bytes|Chars?|File|Lines?) |(?:re)?alloc(?:Shared)?0?(?:Impl)?|recv|repr(?:Discriminant)?|reset |resize(?:Shared)?|runnableExamples |set(?:ControlCHook|CurrentException|FilePos|Frame(?:State)?|GcFrame|Inheritable|Len|StdIoUnbuffered|upForeignThreadGc) |shallow(?:Copy)?|sizeof|slurp|stackTraceAvailable|static(?:Exec|Read)|stdmsg |substr|succ|swap|tearDownForeignThreadGc |to(?:Biggest?(?:Float|Int)|OpenArray(?:Byte)?|U(?:8|16|32))|try(?:Recv|Send) |typeof|unsafe(?:Addr|New)|unsetControlCHook|wasMoved|writeStackTrace |ze(?:64)?|zero(?:Default|Mem)|count(?:down|up)|varargsLen|closureScope |currentSourcePath|disarm|dumpAllocStats|fence|format(?:ErrorIndexBound|FieldDefect) |offsetOf|once|rangeCheck|(?:un)?likely|unown|write(?:Buffer|Bytes|Chars|File|Line)? )\b</string>
```
Comment (Araq) [inline Syntaxes/Nim SCF.tmLanguage:374]: What is this? Remove all of it.

Comment (AmjadHD) [reply Syntaxes/Nim SCF.tmLanguage:374]: It's removed in #170.

**Comment by AmjadHD** [comment]
@Varriount

**Comment by AmjadHD** [comment]
@Varriount.

**Review by Varriount — APPROVED** [review]

**Pull request merged**